yt/yt/client/ya.make

```diff
@@ -198,6 +198,11 @@ SRCS(
     kafka/requests.cpp
 )
 
+CONFIGURE_FILE(
```

Comment:
It seams to be deducted by ya.make from SRCS below. Why do you think that this `CONFIGURE_FILE` is needed?

Reply:
As I mentioned - otherwise resulting file does not materialize after:
ya make --build-all --force-build-depends --replace-result --add-result=.h yt/yt/client/